orchagent/muxorch.cpp

```diff
@@ -431,6 +438,12 @@ bool MuxCable::stateStandby()
     return true;
 }
 
+bool MuxCable::stateCurrent()
```

Comment:
We don't have to go through metric update steps etc from line 468 for this case. I'm not thinking may be a simple check to see if existing and new state is the same, log as NOTICE and return. You've the test to cover that.